File: src/main/resources/scalagen/readSet.scala
val _set = _iprot.readSetBegin()
val _rv = new mutable.HashSet[{{eltType}}]
var _i = 0
while (_i < _set.size) {
  _rv += {
{{eltReader}}
  }
  _i += 1
}
_iprot.readSetEnd()
_rv

File: src/main/resources/javagen/readSet.java
TSet _set = _iprot.readSetBegin();
Set<{{eltType}}> {{name}} = new HashSet<{{eltType}}>(_set.size);
int _i = 0;
while (_i < _set.size) {
  {{eltReader}};
  {{name}}.append(element);
  _i += 1;
}
  _iprot.readSetEnd();